buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -1,5 +1,29 @@
 {
   "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Run AgentTools/Begin.sh",
+      "timeoutInMinutes": 0,
+      "condition": "succeededOrFailed()",
+      "task": {
+        "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
```

Comment:
I believe this is the PS task id, you will need to change it, and the script code will need to be changed.

Reply:
I am not sure what you meant by changing the script code. You mean to update the "script": entry below to use bash commands, right?

Reply:
As per @jcagme suggestion I added a new file, called extensionTask.sh, that can be used in place of begin.sh / end.sh. This bash script receive as parameter the name of the extension task: "begin" or "end".